workbench-core/datasets/src/models/dataSetPermission.ts

```diff
@@ -7,7 +7,9 @@ import { DataSetsAccessLevel } from './dataSetsAccessLevel';
 
 export interface DataSetPermission {
   /** the user or group associated with the access level */
-  subject: string;
+  identity: string;
+  /** the type of subject (group or user) */
+  identityType: string;
```

Comment:
should this be an explicit `type`?

Reply:
I would prefer to leave this open for different possible future implementations and impose our limitation within the plugin.